.github/workflows/e2e.yaml

```diff
@@ -63,4 +63,6 @@ jobs:
       chain-b-tag: '${{ needs.determine-image-tag.outputs.simd-tag }}'
       chain-binary: 'simd'
       # on regular PRs we won't run upgrade tests.
-      test-exclusions: 'TestUpgradeTestSuite,TestGrandpaTestSuite,TestIBCWasmUpgradeTestSuite'
+      # NOTE: we are exluding TestTransferTestSuite as we run this full suite instead of each individual test.
+      test-exclusions: 'TestUpgradeTestSuite,TestGrandpaTestSuite,TestIBCWasmUpgradeTestSuite,TestTransferTestSuite'
```

Comment:
we exclude `TestTransferTestSuite` because we no longer run it as individual tests.